core/services/workflows/engine.go

```diff
@@ -715,10 +720,14 @@ func (e *Engine) worker(ctx context.Context) {
 				continue
 			}
 
+			cma := e.cma.With(eIDKey, executionID)
+			logCustMsg(cma, "starting execution on a trigger event", e.logger)
 			err = e.startExecution(ctx, executionID, resp.Event.Outputs)
 			if err != nil {
 				e.logger.With(eIDKey, executionID).Errorf("failed to start execution: %v", err)
+				logCustMsg(cma, fmt.Sprintf("failed to start execution: %s", err), e.logger)
 			}
+			logCustMsg(cma, "execution started", e.logger)
```

Comment:
@vyzaldysanchez You need an else here: atm if we fail we'll log "failed to start execution" AND "execution started"

Reply:
Oversight on my part, you are right.